newm-server/src/main/kotlin/io/newm/server/database/migration/V26__SongsUpdates.kt

```diff
@@ -3,17 +3,12 @@ package io.newm.server.database.migration
 import org.flywaydb.core.api.migration.BaseJavaMigration
 import org.flywaydb.core.api.migration.Context
 import org.jetbrains.exposed.sql.transactions.transaction
-import java.sql.SQLException
 
 @Suppress("unused")
 class V26__SongsUpdates : BaseJavaMigration() {
     override fun migrate(context: Context?) {
         transaction {
-            try {
-                exec("ALTER TABLE songs RENAME COLUMN ipi TO ipis")
-            } catch (e: SQLException) {
-                println("Column 'ipi' doesn't exist")
-            }
```

Comment:
By accident found this unnecessary try-catch block that apparently I added for debug at some point.